src/Controllers/FrontendVersionController.cs

```diff
@@ -39,8 +40,8 @@ public async Task<ActionResult> Index([FromServices] HttpClient client)
                     },
                     new ()
                     {
-                        Text = "localhost:8080 (local dev)",
-                        Value = "http://localhost:8080/"
+                        Text = $"localhost:{localFrontednPort} (local dev)",
+                        Value = $"http://localhost:{localFrontednPort}/"
```

Comment:
Webpack automatically selects the next available port if 8080 is taken, instead of having this be configurable, I'd much prefer the list to just include 8081 - and possibly 8082 as well (I often run two different versions of app-frontend locally in order to run Cypress tests at the same time as I do other work).